crates/wash-cli/src/dev.rs

```diff
@@ -270,36 +283,26 @@ pub async fn handle_command(
 
     // Since we're using the actor from file on disk, the ref should be the file path (canonicalized) on disk as URI
     let actor_ref = format!("file://{}", artifact_path.display());
-    let actor_id;
-
-    // Attempt to find or create the actor, scaling any existing actors to zero if it exists
-    let inventory = ctl_client.get_host_inventory(&host.id).await.or_else(|e| {
-        bail!(
-            "failed to retrieve host inventory for host [{}]: {e}",
-            &host.id
-        )
-    })?;
-    if let Some(existing_actor) = inventory
-        .actors
-        .into_iter()
-        .find(|a| a.image_ref == Some(actor_ref.clone()))
-    {
-        actor_id = existing_actor.id;
-        scale_actor(&ctl_client, &host.id, &actor_ref, 1, None).await?;
-    } else {
-        // Start the actor for the first time
-        actor_id = start_actor(StartActorArgs {
-            ctl_client: &ctl_client,
-            host_id: &host.id,
-            actor_ref: &actor_ref,
-            count: 1,
-            skip_wait: false,
-            timeout_ms: None,
-        })
-        .await?
-        .actor_id
-        .ok_or_else(|| anyhow!("failed to do thing"))?;
-    }
+    // Since the only restriction on actor_id is that it must be unique, we can just use the artifact path as the actor_id
+    // to ensure uniqueness
+    let actor_id = artifact_path.display().to_string();
```

Comment:
Doesn't the actor_id need to be NATS subject compliant?

Reply:
It does, good point. Worth a sanitization pass

Reply:
I tried to validate component IDs all over the CLI with a value parser